cmd/config/cache/config.go

```diff
@@ -152,3 +154,13 @@ func parseCacheExcludes(excludes string) ([]string, error) {
 
 	return excludesSlice, nil
 }
+
+func parseCacheCommitMode(commitStr string) (bool, error) {
+	switch strings.ToLower(commitStr) {
+	case "async":
+		return true, nil
+	case "sync":
+		return false, nil
+	}
```

Comment:
I would rather call it 'writeback' and 'writethrough' since that reflects the actual caching terms we are implementing.

Reply:
this is done.